### DISCUSSION TITLE
Lasy RIO

### DISCUSSION THREAD

**Discussion by DagiDCS** [aronCiucu/DCSTheWay#172] [Ideas]
Can we have this for the F14 Tomcat Rio too, so us lasy RIO's can tiktaktoe the waypoints subafast?

**Comment by DagiDCS** [comment]
Still no Lazy RIO :(

**Reply by ozdeadmeat** [reply]
I think what he is asking for is a method of having a player RIO utilze TheWay. Is that possible?
